src/client/opamConfigCommand.ml

```diff
@@ -71,7 +71,7 @@ let rec print_env output = function
   | [] -> ()
   | (k, v, comment) :: r ->
     if OpamConsole.verbose () then
-      OpamStd.Option.iter (Printf.ksprintf output ": %s;\n") comment;
+      Stdlib.Option.iter (Printf.ksprintf output ": %s;\n") comment;
```

Comment:
Side note for reviewers: explicit uses of `Stdlib` is required in `opam-client` and `opam-solver` because `extlib` (used via `dose3`) shadows the standard library module